metadata-ingestion/tests/integration/snowflake-beta/snowflake_beta_golden.json

```diff
@@ -915,7 +915,7 @@
     "changeType": "UPSERT",
     "aspectName": "datasetProperties",
     "aspect": {
-        "value": "{\"customProperties\": {}, \"name\": \"TABLE_10\", \"qualifiedName\": \"test_db.test_schema.table_10\", \"description\": \"Comment for Table\", \"tags\": []}",
+        "value": "{\"customProperties\": {}, \"externalUrl\": \"https://app.snowflake.com/us-west-2/abc12345/#/data/databases/TEST_DB/schemas/TEST_SCHEMA/table/TABLE_10/\", \"name\": \"TABLE_10\", \"qualifiedName\": \"test_db.test_schema.table_10\", \"description\": \"Comment for Table\", \"tags\": []}",
```

Comment:
Minor Since we already have context, let's add some test cases for the private link urls as well:
(https://app.{region_id}.privatelink.snowflakecomputing.com)

It seems that everything after the end of the path will remain the same. We can confirm via email with the Snowflake folks)